pkg/kubectl/resource/visitor.go

```diff
@@ -396,19 +396,21 @@ func ExpandPathsToFileVisitors(mapper *Mapper, paths string, recursive bool, ext
 			if path != paths && !recursive {
 				return filepath.SkipDir
 			}
-			return nil
```

Comment:
@metral actually any code change here is not necessary. We have already consider multi level directory recursive.